hw/ip_templates/racl_ctrl/rtl/racl_ctrl.sv.tpl

```diff
@@ -123,28 +123,29 @@ module ${module_instance_name} import ${module_instance_name}_reg_pkg::*; #(
   //////////////////////////////////////////////////////////////////////////////////////////////////
   // Error handling
   //////////////////////////////////////////////////////////////////////////////////////////////////
-  //////////////////////////////////////////////////////////////////////////////
 
   // A RACL error can only happen for one IP at a time in one RACL domain. Therefore, it is
   // safe to OR all RACL error bits together and no arbitration is needed. This is true also
   // for the corresponding RACL role or Write/Read information.
+  `ASSERT(OneHotRaclError_A, $onehot0(racl_error_i))
+
   logic racl_error;
   assign racl_error = |racl_error_i;
 
   top_racl_pkg::racl_role_t racl_error_role;
   top_racl_pkg::ctn_uid_t racl_error_ctn_uid;
-  logic racl_error_write_read;
+  logic racl_error_read_not_write;
```

Comment:
I don't think this change helps clarify what this bit means: 1 == read, 0 == write? If so, maybe call it read_access?

Reply:
Exactly. This encoding is documented in `top_racl_pkg.sv` at the struct definition. I renamed the field to access type to remove the confusion.